### ISSUE TITLE
PyMakr IDE deprecated, use PyMakr plugin instead

### ISSUE THREAD

**Issue by originalfoo** [ttn-liv/devices#2]
In relation to [this](https://github.com/ttn-liv/devices/wiki/Getting-started-with-the-PyCom-LoPy#simple-micropython-tests), the PyMakr IDE is deprecated.

Instead, PyCom recommend the PyMakr plugin for third party IDEs. Currently only GitHub Atom is supported but they are working on more.

https://docs.pycom.io/pycom_esp32/pycom_esp32/toolsandfeatures.html#pymakr-plugin

**Comment by ajlennon** [comment]
I found PyMakr to be pretty flakey to be honest.

**Comment by originalfoo** [comment]
Yup, I had lots of pain trying to get it installed in Atom - ended up having to reinstall Node JS. Since then, it's been working reliably. I think the wiki page should still link to the plugin though, as Pycom have stated that is their future direction (plugins, not dedicated IDE).

**Comment by ajlennon** [comment]
For me I don't mind if something is deprecated as long as it works.

I'd prefer an easy installation route to get started rather than the pain of Atom.
